pkgs/development/python-modules/pyexcel-xls/default.nix

```diff
@@ -30,6 +32,10 @@ buildPythonPackage rec {
     mock
   ];
 
+  postPatch = ''
+    sed -i 's/xlrd<2/xlrd<3/g' setup.py
```

Comment:
You can also use:

```suggestion
    substituteInPlace setup.py --replace "xlrd<2" "xlrd<3"
```

Reply:
Is that generally preferred instead of using sed?

Reply:
Yes.

Reply:
OK, I changed it now.
The reason I was preferring sed was that I have been noticing that substitueInPlace is orders of magnitude slower on a large scale.
But substitueInPlace is more readable which is more important here I guess.